activestorage/test/controllers/representations/proxy_controller_test.rb

```diff
@@ -18,10 +18,7 @@ class ActiveStorage::Representations::ProxyControllerWithVariantsTest < ActionDi
 
     assert_response :ok
     assert_match(/^attachment/, response.headers["Content-Disposition"])
-
-    image = read_image(@blob.variant(@transformations))
-    assert_equal 100, image.width
-    assert_equal 67, image.height
+    assert_equal @blob.representation(@transformations).download, response.body
```

Comment:
Since this test specifically targets variants (not previews), let's use `variant` instead of `representation` here.

```suggestion
    assert_equal @blob.variant(@transformations).download, response.body
```

I know they are equivalent, but using `variant` feels more correct here.  It also feels slightly more like double checking since, presumably, it hits a different code path [than `ProxyController` does](https://github.com/rails/rails/blob/8b34759596efe31909a3cf632903c6d1a2479b62/activestorage/app/controllers/active_storage/representations/base_controller.rb#L14).

Reply:
Alright let's do this.

If I undrstand correctly, I will also change from `.representation` to `.preview` in the preview tests!

Please let me know if I got it wrong